docs/exts/taglist.py

```diff
@@ -0,0 +1,112 @@
+from sphinx.util import logging
+from docutils.parsers.rst import Directive
+from docutils import nodes
+from sphinx.environment import NoUri
+
```

Comment:
Please add some information on how this extension should be used as a top comment somewhere here.

Reply:
Fixed